packages/core-types/tests/integration/components/field-editors/integer-editor-test.js

```diff
@@ -0,0 +1,57 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, triggerEvent, waitFor } from '@ember/test-helpers';
+import Object from '@ember/object';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | field editors/integer editor', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    this.setProperties({
+      model: Object.create({ rating: 3 }),
+      onchange() {}
+    });
+    await render(hbs`
+      {{field-editors/integer-editor
+        content=model
+        field="rating"
+        onchange=onchange
+      }}`);
+    assert.dom('input[type=number]').hasValue('3', 'number input has correct value');
+    assert.dom('input').isNotDisabled('input is not disabled');
+  });
+
+  test('it updates the value correctly', async function(assert) {
+    this.setProperties({
+      model: Object.create({ rating: 3 }),
+      onchange() {}
+    });
+    await render(hbs`
+      {{field-editors/integer-editor
+        content=model
+        field="rating"
+        onchange=onchange
+      }}`);
+    this.set('model.rating', 5);
+    await triggerEvent('.field-editor > input', 'input');
```

Comment:
I'm not sure I understand what is being tested here. Are you testing a user input? maybe the `fillIn()` or `typeIn()` test helpers might be better suited here?

Reply:
True, I was so focused on triggering an `input` event (since that's what the component reacts to) that I lost sight of the fact that `fillIn` does the same and reads better.